pkg/api/v1beta1/defaults.go

```diff
@@ -102,3 +108,14 @@ func init() {
 		},
 	)
 }
+
+// With host networking default all container ports to host ports.
+func defaultHostNetworkPorts(containers *[]Container) {
+	for i := range *containers {
+		for j := range (*containers)[i].Ports {
+			if (*containers)[i].Ports[j].ContainerPort == 0 {
+				(*containers)[i].Ports[j].ContainerPort = (*containers)[i].Ports[j].HostPort
```

Comment:
@ArtfulCoder Sorry, I'm confused. There is no ContainerIP field.

ContainerPort is required, so it cannot be defaulted.